### ISSUE TITLE
[phpcs] Not in project root

### ISSUE THREAD

**Issue by bytestream** [DaveLiddament/sarb#70]
Long directory paths in phpcs text reports are truncated
```
FILE: ...eran/test/foo/bar/quz/test.php
--------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
```

Links:
* https://github.com/squizlabs/PHP_CodeSniffer/blob/9c90cbfafe32be55a0b00e2bda4035d48f76a7d9/src/Reports/Full.php#L85
* https://github.com/squizlabs/PHP_CodeSniffer/wiki/Configuration-Options#setting-the-default-report-width

I'd be grateful if the [PHPCS documentation](https://github.com/DaveLiddament/sarb/blob/master/docs/PhpCodeSniffer.md) was updated to account for this:
```
vendor/bin/phpcs src/ > /tmp/phpcs.txt
```
becomes
```
vendor/bin/phpcs --report-width=auto src/ > /tmp/phpcs.txt
```

Perhaps with a note explaining what `--report-width` is for in case the user has to run a different command, for example I used a modified command:
```
./vendor/bin/phpcs --report-width=auto --standard=phpcs.xml . > /tmp/phpcs.txt
```

----

**Edit**

Alternative workaround - use the undocumented JSON parser https://github.com/DaveLiddament/sarb/pull/60:
```
$ ./vendor/bin/phpcs --report=json --standard=phpcs.xml . > /tmp/phpcs.txt
$ ./vendor/bin/sarb create-baseline /tmp/phpcs.txt phpcs.baseline phpcodesniffer-json
Baseline created
Errors in baseline 12617
```

**Comment by DaveLiddament** [comment]
Hi @bytestream 

Thanks for raising an issue and both of these suggestions for fixes. I will update documentation in the next couple of days. 

Really appreciate your input to helping improve SARB.

**Comment by DaveLiddament** [comment]
Fixed in #80

**Comment by bytestream** [comment]
Thanks!
